Lib/tkinter/test/test_tkinter/test_misc.py

```diff
@@ -156,6 +156,24 @@ def callback():
         with self.assertRaises(tkinter.TclError):
             root.tk.call('after', 'info', idle1)
 
+    def test_winfo_rgb(self):
+        root = self.root
+        rgb = root.winfo_rgb
+
+        # Color name.
+        self.assertEqual(rgb('red'), (65535, 0, 0))
+        # #RGB
+        self.assertEqual(rgb('#f00'), (65535, 0, 0))
+        # #RRGGBB
+        self.assertEqual(rgb('#ff0000'), (65535, 0, 0))
```

Comment:
It is not clear from what lower bits are taken. Use examples with not all bits set or reset, e.g. `'#123456789'`. And represent expected values as hexadecimals.